README.md

```diff
@@ -31,8 +31,9 @@ let g:keepeye_autostart = v:true                       " launch at vim startup
 let g:keepeye_callback = 'keepeye#Callback'            " customize the callback
 let g:keepeye_message = 'SAVE YOUR EYES, TAKE A BREAK' " set the message to show
 let g:keepeye_system_bell = v:false                    " enable the bell sound (1)
+let g:keepeye_bell_player = 'mocp -l'                  " set a custom to ring the bell"
 let g:keepeye_system_notification = v:false            " enable the system notification (2)
 let g:keepeye_timer = 1500                             " set the work time, 25min by default
 ```
-1. [mpv](https://mpv.io/) is required for this feature
+1. [mpv](https://mpv.io/) is the default player for this feature
```

Comment:
Let's put 3 players: mpv, cvlc and moc (Players available: mpv, cvlc and moc + link)